src/ChatItem.tsx

```diff
@@ -0,0 +1,70 @@
+import { Chat, User } from './Interfaces';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  display: flex;
+  gap: 10px;
+  width: 100%;
+  justify-content: ${({ isUser }: { isUser: boolean }) =>
+    isUser ? 'flex-end' : 'flex-start'};
+`;
+const ContentWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+`;
+const ChatWrapper = styled.div`
+  display: flex;
+  align-items: flex-end;
+  gap: 5px;
+`;
+const ChatValue = styled.div`
+  background-color: ${({ isUser }: { isUser: boolean }) =>
+    isUser ? '#ECECEC' : 'white'};
+  padding: 10px;
+  border-radius: 10px;
+  word-break: break-all;
+`;
```

Comment:
이부분은 interface를 사용하면 위의 동일한 isUser가 props로 넘어오는 것에 대해 나름 깔끔하게 처리할 수 있을 것 같아요
```suggestion
interface CVProps {
  isUser: boolean;
}
const ChatValue = styled.div<CVProps>`
  background-color: ${(props) => props.isUser ?  '#ECECEC' : 'white'};
  padding: 10px;
  border-radius: 10px;
  word-break: break-all;
`;
```